lib.php

```diff
@@ -28,7 +28,6 @@
 defined("MOODLE_INTERNAL") || die();
 define("COURSEFEEDBACK_DEFAULT", "DEFAULT");
 define("COURSEFEEDBACK_ALL", "ALL");
```

Comment:
While you are at it, you might as well remove the `MOODLE_INTERNAL` check. It is not necessary in files without side effects. (See [docs](https://moodledev.io/general/development/policies/codingstyle#require--include)) And this file only defines constants and functions.

Also defining scalar constants is more readable IMO using the more modern `const` keyword instead of the old `define` function.

```suggestion
const COURSEFEEDBACK_DEFAULT = "DEFAULT";
const COURSEFEEDBACK_ALL = "ALL";
```